app/views/users/_user.html.erb

```diff
@@ -1,5 +1,6 @@
 <tr>
   <td><%= link_to user.name, user %></td>
+  <td><%= user.email %></td>
```

Comment:
Given that an email search is only going to return one User, and the limited real-estate on the table view,  I'm thinking that we don't display the email address in the table view.